includes/polling/os/unifi.inc.php

```diff
@@ -1,9 +1,11 @@
 <?php
 
-if ($sysmodel = snmp_get($device, 'unifiApSystemModel.0', '-Osqnv', 'UBNT-UniFi-MIB')) {
-    $hardware = $sysmodel;
-}
-
-if ($sysver = snmp_get($device, 'unifiApSystemVersion.0', '-Osqnv', 'UBNT-UniFi-MIB')) {
-    $version = $sysver;
+if ($data = snmp_get_multi($device, 'unifiApSystemModel unifiApSystemVersion', '-OUQs', 'UBNT-UniFi-MIB')) {
+    $hardware = $data[0]['unifiApSystemModel'];
+    $version = $data[0]['unifiApSystemVersion'];
+} elseif ($data = snmp_get_multi($device, 'dot11manufacturerProductName dot11manufacturerProductVersion', '-OQUs', 'IEEE802dot11-MIB')) {
```

Comment:
This needs to be two separate snmp_getnext() calls.  But one can be inside the if statement ;)

Reply:
Ah yep, not a problem. This does make the code a bit messier - snmp_get_multi() returns a nice convenient array indexed by OID, while snmp_getnext() might return from an OID we didn't ask for & is a single string.